src/main/java/timl/TimL.java

```diff
@@ -0,0 +1,104 @@
+package timl;
+
+import timl.Exeptions.EmptyException;
+import timl.Exeptions.TimException;
+import timl.task.Task;
+import timl.task.TaskManager;
+import timl.utility.Printer;
+import timl.utility.TextParser;
+
+import java.util.Scanner;
+import java.lang.String;
+
+public class TimL {
+    public static void respondToCommand(String response, Task[] list){
```

Comment:
this function is quite long (30+ LOC). this can affect redability. consider abstracting out parts of the code into relevant functions

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods